modules/authorization/src/Authorization.ts

```diff
@@ -125,6 +127,18 @@ export default class Authorization extends ManagedModule<Config> {
     callback(null, {});
   }
 
+  async createRelations(
+    call: GrpcRequest<CreateRelationsRequest>,
+    callback: GrpcResponse<Empty>,
+  ) {
+    const subject = call.request.subject;
+    const relation = call.request.relation;
+    const resources = call.request.resources;
+    await this.relationsController.checkRelations(subject, relation, resources);
```

Comment:
Why is this explicitly called outside `createRelations()` in the first place?
Doesn't seem to be used anywhere else either.
Imo, make it private or integrate directly back into `createRelations()`.